src/nyc_trees/apps/home/routes.py

```diff
@@ -19,3 +21,7 @@
 
 training_list_page = route(GET=do(render_template('home/training.html'),
                                   v.training_list_page))
+
+groups_to_follow = route(GET=do(render_template('home/groups_to_follow.html'),
+                                update({'chunk_size': 2}),
```

Comment:
I think creating a new view would still be simpler than the `update` decorator but I would get another opinion.